tests/introspection/iOS/iOSApiSelectorTest.cs

```diff
@@ -106,7 +106,12 @@ protected override bool Skip (Type type)
 			// iOS 10 - this type can only be instantiated on devices, but the selectors are forwarded
 			//  to a MTLHeapDescriptorInternal and don't respond - so we'll add unit tests for them
 			case "MTLHeapDescriptor":
-				return Runtime.Arch == Arch.DEVICE;
+			case "MTLLinkedFunctions":
+			case "MTLRenderPipelineDescriptor":
+			case "MTLTileRenderPipelineDescriptor":
+			case "MTLRasterizationRateLayerDescriptor":
+			case "MTLRasterizationRateMapDescriptor":
+				return Runtime.Arch == Arch.SIMULATOR;
```

Comment:
Why did this change from `== Arch.DEVICE` to `== Arch.SIMULATOR`? Was that intentional?

Reply:
Yes. The skip should be when we are on simulator, not the other way around. I guess it was a mistake in a previous change